gazelle/generate.go

```diff
@@ -213,7 +214,6 @@ func (lang *JS) GenerateRules(args language.GenerateArgs) language.GenerateResul
 					imports:  kindImports,
 				}, jsConfig)
 				if !jsConfig.Quiet && len(moduleRules) > 1 {
-					log.Print(Warn("[WARN] disjoint module %s", args.Rel))
```

Comment:
Is this a debug change?

Reply:
Hmm, this generated a lotta noise I will say. I can use the Quiet flag.